src/components/Dashboard/DashboardPieChart.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <el-card class="card">
+    <el-row>
+      <el-col :span="11">
+        <div class="list_crypto">
+          <div class="list_crypto-content">
```

Comment:
🎨 Layout is still broken a bit. I think flexbox will solve this :)

width=1200
![electron](https://user-images.githubusercontent.com/1365915/42678006-89d282fc-86b8-11e8-84a4-2b08510f00d5.png)
width=1600
![electron](https://user-images.githubusercontent.com/1365915/42677989-7a6500a6-86b8-11e8-978a-d7eb7759c61f.png)

Reply:
I think the cause is that `list_crypto-content-item` can go multiline like:
![electron](https://user-images.githubusercontent.com/1365915/42797988-dccd205e-89cc-11e8-8e87-8459432163c7.png)

This is because `list_crypto-content-item` is too narrow to contain a long currency name, so - what about setting something like `flex: 1 0 5rem` to `list_crypto-content-item` instead of width?